docs/pages/web/iconbuttonfloating.js

```diff
@@ -156,42 +109,53 @@ If IconButtonFloating is used as a control button to show/hide a Popover-based c
-          IconButtonFloating is lifted off of the background with shadows built-in ($elevation-floating). It must be absolutely positioned to achieve the floating effect.
-
-          In dark mode, the elevation is achieved with colors instead of shadows ($color-background-elevation-floating). See [elevation guidelines](/foundations/elevation) for reference. `}
-        />
+          description="IconButtonFloating size is only available in a single size, 56px (the equivalent of [IconButton's size='xl'](/web/iconbutton#Size)). Keeping the size consistent will promote a cohesive Pinner experience and avoid usability issues. "
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={doForScroll} name="Scroll Placement example" />}
+          />
+        </MainSection.Subsection>
         <MainSection.Subsection
+          columns={2}
           title="Placement"
           description={`
-          IconButtonFloating is always placed along the bottom of the screen. A consistent position improves discoverability when IconButtonFloating appears across a responsive range of surfaces. For example, an IconButtonFloating used to open resources should remain in the same spot on the page across surfaces. In most cases, only one IconButtonFloating should be present on a screen. The exception is using a centered IconButtonFloating as a primary action, like board creation.
+          IconButtonFloating is always placed along the bottom of the screen. A consistent position improves discoverability when IconButtonFloating appears across a responsive range of surfaces. For example, an IconButtonFloating used to open resources should remain in the same spot on the page across surfaces. In most cases, only one IconButtonFloating should be present on a screen. The exception is using a centered IconButtonFloating as a primary action, like Pin or board creation.
 
           **Bottom edge placement**
           IconButtonFloating should typically be placed in the bottom edge of the screen (bottom right for LTR languages, and bottom left for RTL languages). This applies to supportive actions, such as opening related content and resources.
 
           **Centered placement**
           Use a centered placement when leading Pinners to an action or task that should remain present when scrolling, such as creating a new board.
 `}
-        />
+        >
+          <MainSection.Card
+            title="Bottom edge placement"
+            cardSize="md"
+            sandpackExample={<SandpackExample code={doForScroll} name="Scroll Placement example" />}
+          />
+          <MainSection.Card
+            title="Centered placement"
+            cardSize="md"
+            sandpackExample={<SandpackExample code={a11y} name="A11Y Placement example" />}
+          />
+        </MainSection.Subsection>
       </MainSection>
       <MainSection name="Writing">
         <MainSection.Subsection columns={2}>
```

Comment:
nit: I don't think the comma is needed in "Button allows users to take _actions and_ make choices using text labels…"